### ISSUE TITLE
[Feature Request] Aarch64 Build

### ISSUE THREAD

**Issue by Saroufim** [flathub/de.wger.flutter#2]
It would be nice to have an Aarch64 build especially since the app is already optimized for different screen sizes.

**Comment by rolandgeider** [comment]
yes, this is definitely coming as soon as I find some time to build it on github actions

**Comment by CocolinoFan** [comment]
Yes please. Can't install the app on my Librem 5.

**Comment by rolandgeider** [comment]
At the moment arm runners are only available for enterprise plans, but it seems this is coming to open source projects as well. As soon as this is available, we'll update

https://github.blog/news-insights/product-news/arm64-on-github-actions-powering-faster-more-efficient-build-systems/
